packages/api-explorer/src/ResponseExample.jsx

```diff
@@ -41,15 +42,30 @@ function getReactJson(example, responseExampleCopy) {
   );
 }
 
-function showMediaTypes(example, setResponseMediaType, responseMediaType) {
+/**
+ * Determine if we should render a dropdown of available media types for a given response example.
+ *
+ * @param {object} example
+ * @param {function} setResponseMediaType
+ * @param {string} responseMediaType
+ * @param {boolean} [nestInTabberBarDivClass=false] If `true`, the dropdown will be rendered within
+ *    a div that has the `tabber-bar` class.
+ * @returns jsx
+ */
+function showMediaTypes(
+  example,
+  setResponseMediaType,
+  responseMediaType,
+  nestInTabberBarDivClass = false,
+) {
   const mediaTypes = example.languages;
   if (mediaTypes.length <= 1) return false;
 
   let responseMediaTypeCopy = responseMediaType;
   if (!responseMediaTypeCopy && mediaTypes[0]) responseMediaTypeCopy = mediaTypes[0].languages;
 
   return (
-    <div>
+    <div className={nestInTabberBarDivClass ? 'tabber-bar' : ''}>
```

Comment:
This change is in here because with some CSS that we added for the multiple media type dropdown adds top-padding if `tabber-bar` is present. Problem is that if you don't have multiple media types to show, we were still showing that padding because the `<div class="tabber-bar"></div>` element was present and empty. 

Now if we aren't going to be showing a dropdown, we won't add the `tabber-bar` div into the DOM.

Reply:
Ohh word, good catch. I think I tried to solve this with a fancy-ass `:empty` selector, but ran in to some road blocks around the markup structure! This looks like a good solution.